frontend/src/components/styles/ProfileCard.scss

```diff
@@ -0,0 +1,31 @@
+@import "./../../index.scss";
+
+.profileCard {
+  width: 300px;
+  height: 380px;
+  border: 5px solid $mainBrandColor;
+  position: relative;
+
+  .gradientName {
+    position: absolute;
+    top: 250px;
+    left: 0;
+    height: 50px;
+    width: 100%;
+    margin: 0;
+    background-image: linear-gradient(to bottom, #00b0f000, #00b0f0);
```

Comment:
You can use variable instead of hex color

Reply:
done